src/Build/Resources/Strings.resx

```diff
@@ -527,7 +527,7 @@
     </comment>
   </data>
   <data name="InvalidFunctionStaticMethodSyntax" UESanitized="false" Visibility="Public">
-    <value>MSB4186: Invalid static method invocation syntax: "{0}". {1} Static method invocation should be of the form: $([FullTypeName]::Method()), e.g. $([System.IO.Path]::Combine(`a`, `b`)). </value>
+    <value>MSB4186: Invalid static method invocation syntax: "{0}". {1} Static method invocation should be of the form: $([FullTypeName]::Method()), e.g. $([System.IO.Path]::Combine(`a`, `b`)). Ensure that all parameters are defined, are of the correct type, and are specified in the right order.</value>
```

Comment:
`Ensure` seems too prophetic, maybe use `Check`? For example, what does it mean for all parameters to be defined? MSBuild treats undefined properties and items as empty properties and items, so practically msbuild does not have undefined variables. So undefined variables could be totally fine for the static calls which accept empty strings.

Reply:
@cdmihai: Should I change it to `Check`?

Reply:
Yes please. Then trigger a build to update the xlf please. Looks good to merge after that.